orbit/db2.py

```diff
@@ -63,11 +86,10 @@ def del_by_username(username: str) -> list[Self]:
         return list(res)
 
 
-class NewUser(BaseModel):
-    id = IntegerField(primary_key=True)
-    student_id = CharField(unique=True)
-    username = CharField(unique=True)
+class Registration(BaseModel):
```

Comment:
random churn on the order of these fields and name of class is distracting, if you want them to be in the other order, it should be melded into the commit where you first introduced this code